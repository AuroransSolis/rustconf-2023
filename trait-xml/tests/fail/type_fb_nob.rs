trait_xml::trait_xml! {
    <trait>
        <name>Foo</name>
        <bounds>
            <type>
                <name>Bar</name>
                <for-bound>
                    <lifetime>'baz</lifetime>
                </for-bound>
            </type>
        </bounds>
    </trait>
}

fn main() {}
