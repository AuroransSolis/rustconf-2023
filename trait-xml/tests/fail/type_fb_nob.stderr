error: error parsing `for` bound: no bound provided. caller: `::trait_xml::trait_xml_parse_type`
  --> tests/fail/type_fb_nob.rs:1:1
   |
1  | / trait_xml::trait_xml! {
2  | |     <trait>
3  | |         <name>Foo</name>
4  | |         <bounds>
...  |
12 | |     </trait>
13 | | }
   | |_^
   |
   = note: this error originates in the macro `$crate::trait_xml_parse_for_bound` which comes from the expansion of the macro `trait_xml::trait_xml` (in Nightly builds, run with -Z macro-backtrace for more info)
